tests/python/builtin.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+import unittest
+import subprocess
+
+
+class TestBuiltin(unittest.TestCase):
+
+    def run_test(self, cmd, expected):
+        p = subprocess.Popen(
+            [cmd], shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        self.assertRegexpMatches(p.stdout.readline(), expected)
+
+    def test_pid(self):
+        cmd = """test=pid; ../.././build-release/src/bpftrace -e \
```

Comment:
Extra dot in here
Also, I think the path to the bpftrace executable should be configurable - not everyone uses the docker build scripts